src/components/PaginatedTable/TableChunk.tsx

```diff
@@ -93,8 +94,12 @@ export const TableChunk = typedMemo(function TableChunk<T, F>({
 
     React.useEffect(() => {
         if (currentData && isActive) {
-            const {total = 0, found = 0} = currentData;
-            onDataFetched(total, found);
+            onDataFetched({
+                ...currentData,
+                data: currentData.data as T[],
+                found: currentData.found || 0,
+                total: currentData.total || 0,
+            } as PaginatedTableData<T>);
```

Comment:
`as PaginatedTableData<T>` is redundant, `data: currentData.data as T[]` is enough